package.yaml

```diff
@@ -8,6 +8,7 @@ copyright: "AllRightsReserved"
 verbatim:
   data-files: |
     ts/**/*.ts
+    ts/**/*.json
```

Comment:
This is using cabal globs verbatim and that doesn't allow file globs without extensions, see https://cabal.readthedocs.io/en/stable/cabal-package.html#pkg-field-data-files.